eZ/Publish/API/Repository/Tests/SearchServiceFieldFiltersTest.php

```diff
@@ -29,14 +29,34 @@ public function setUp()
     {
         $setupFactory = $this->getSetupFactory();
 
-        if ( !$setupFactory instanceof LegacyElasticsearch )
+        if ( $setupFactory instanceof LegacySolr )
         {
-            $this->markTestIncomplete( "ATM implemented only for Elasticsearch storage" );
+            $this->markTestIncomplete( "Not implemented for Solr Search Engine" );
         }
 
         parent::setUp();
     }
 
+    protected function supportsFilteringFullText()
```

Comment:
Changed in https://github.com/ezsystems/ezpublish-kernel/commit/6778d580e834b26bdfabe37b6af85dea595fcb76.